tests/utils/model_configs.py

```diff
@@ -682,7 +683,7 @@ def _update_and_add_testing_config(
     groups={
         ModelTestingGroup.basic: ModelTestingGroupAction.normal,
         ModelTestingGroup.checkpoint: ModelTestingGroupAction.normal,
-        ModelTestingGroup.convert: ModelTestingGroupAction.normal,
+        ModelTestingGroup.convert: ModelTestingGroupAction.broken,
```

Comment:
is this because of some weird triton or cuda errors? that's a version conflict...

Reply:
This one was broken already, let's just drop the whole config since we don't need it.